edgedb/datatypes/record_desc.c

```diff
@@ -30,6 +30,7 @@ record_desc_dealloc(EdgeRecordDescObject *o)
     PyObject_GC_UnTrack(o);
     Py_CLEAR(o->index);
     Py_CLEAR(o->names);
+    Py_CLEAR(o->get_dataclass_fields);
```

Comment:
Do we need to visit it in `record_desc_traverse`?

Reply:
OK, this is a function.. the name is confusing. I suggest renaming to `get_dataclass_fields_func`